initialdata/pg_class.csv

```diff
@@ -0,0 +1,420 @@
+oid,relname,relnamespace,reltype,reloftype,relowner,relam,relfilenode,reltablespace,relpages,reltuples,relallvisible,reltoastrelid,relhasindex,relisshared,relpersistence,relkind,relnatts,relchecks,relhasrules,relhastriggers,relhassubclass,relrowsecurity,relforcerowsecurity,relispopulated,relreplident,relispartition,relrewrite,relfrozenxid,relminmxid,relacl,reloptions,relpartbound
```

Comment:
It would be better to add a script for generating these files:

```sh
CONTAINER_ID=$(docker run --rm -d -e POSTGRES_PASSWORD=postgres postgres)
docker exec -i $CONTAINER_ID psql -U postgres -c '\COPY pg_class TO STDOUT (FORMAT CSV)' > pg_class.csv
...
docker kill $CONTAINER_ID
```

Reply:
This approach may not work with cloud vendors in China, as the machine might not be able to connect to Docker Hub.

Reply:
I mean a script for maintainers to regenerate these files easily.

Reply:
Got it.I'll leave a script with README later.

Reply:
fixed